config/routes.rb

```diff
@@ -1,4 +1,7 @@
 Rails.application.routes.draw do
+  get 'home'  => 'static_pages#home'
```

Comment:
sử dụng cú pháp `get "/home", to: "static_pages#home"` em nhé
Hơn nữa, e dùng " thay vì ' trong các file nhé